### PR TITLE
[Hotfix] Trimmed whitepace from user registration fields inputs

### PR DESCRIPTION
**Pull request by bendozy**

### CODE REVIEW

File: app/Http/Controllers/HomeController.php [hunk 1]
```diff
@@ -71,12 +71,13 @@ public function firstProfile(ProfileRequest $request, UserRepository $userReposi
...
-        $user->firstname = $request ['firstname'];
-        $user->lastname = $request ['lastname'];
-        $user->location = $request ['location'];
-        $user->gender = $request ['gender'];
-        $user->best_food = $request ['best_food'];
-        $user->image_uri = $this->getAvatarUrl($user);
+        $user->firstname = trim($request ['firstname']);
+        $user->lastname = trim($request ['lastname']);
+        $user->location = trim($request ['location']);
+	    $user->about = trim($request ['about']);
```
Comment (unicodeveloper) [inline app/Http/Controllers/HomeController.php]: Adjust the indentation

File: app/Http/Controllers/Auth/AuthController.php [hunk 2]
```diff
@@ -2,178 +2,196 @@
...
+			'password' => 'required|confirmed|min:8'
+		]);
+	}
+
+	public function getSocialPassword(Request $request)
+	{
+		if(! $request->session()->has('socialUser')) {
+			return redirect()->intended('/login');
+		}
+		return view('auth.set_social_password');
```
Comment (unicodeveloper) [inline app/Http/Controllers/Auth/AuthController.php:147]: There should be space between line 146 and 147

File: app/Http/Controllers/Auth/AuthController.php [hunk 3]
```diff
@@ -2,178 +2,196 @@
...
+			'name' => 'required|max:255|unique:users,username|min:3'
+		]);
+
+		if($validation->fails()) {
+			return redirect()->back()->withInput()->withErrors($validation->errors());
+		} else{
+			$user_array = array(
+				'email' => $request->session()->get('socialUser')->getEmail(),
+				'name' => $request->all() ['name'],
+				'password' => $request->all() ['password']
```
Comment (unicodeveloper) [inline app/Http/Controllers/Auth/AuthController.php]: Just use `$request->input` instead you are trying to fetch a single value

File: app/Http/Controllers/Auth/AuthController.php [hunk 4]
```diff
@@ -2,178 +2,196 @@
...
+	 * Handle a registration request for the application.
+	 *
+	 * @param \Illuminate\Http\Request $request
+	 *
+	 * @return \Illuminate\Http\Response
+	 */
+	public function postRegister(Request $request)
+	{
+
+		$request->all()['name'] = trim($request->all()['name']);
```
Comment (unicodeveloper) [inline app/Http/Controllers/Auth/AuthController.php]: Use `$request->input` instead when you are trying to fetch a single value

**Comment by bendozy** [comment]
Issues have been fixed

**Pull request merged**